src/lib/menu/menu-trigger.ts

```diff
@@ -70,6 +70,7 @@ export class MdMenuTrigger implements AfterViewInit, OnDestroy {
     if (!this._menuOpen) {
       this._createOverlay();
       this._overlayRef.attach(this._portal);
+      this._overlayRef.backdropClick().subscribe(() => this.closeMenu());
```

Comment:
Since the backdrop doesn't have anything to do with positioning, I'd do this as a unit tests. We generally want to prefer capturing behaviors with unit tests because they are so much faster.

@fxck I would use `.first()`, but yeah that should be the the case.